src/main/java/datawave/data/type/OneToManyNormalizerType.java

```diff
@@ -7,4 +7,6 @@ public interface OneToManyNormalizerType<T extends Comparable<T>> extends Type<T
     List<String> normalizeToMany(String in);
     
     List<String> getNormalizedValues();
+    
+    boolean expandAtQueryTime();
```

Comment:
If we have any other implementations of this, then perhaps a default implementation?  If no other implementations then you can ignore this.